src/server/gc_worker/compaction_filter.rs

```diff
@@ -180,13 +199,23 @@ impl CompactionFilterFactory for WriteCompactionFilterFactory {
         };
 
         let db = Arc::clone(&gc_context.db);
+        let orphan_versions_handler = gc_context.orphan_versions_handler.clone();
 
         debug!(
             "creating compaction filter"; "feature_enable" => enable,
             "skip_version_check" => skip_vcheck,
             "ratio_threshold" => ratio_threshold,
         );
 
+        if db
+            .get_property_int("rocksdb.is-write-stalled")
```

Comment:
seems there is no property called that, but a property named `is-write-stopped`. I suggest leveraging the hook `on_stall_conditions_changed` to record a global stall status in TiKV.

Reply:
It's a new added property.

Reply:
oh, I see

Reply:
should consider `is-write-stopped` too

Reply:
I guess `stalled` contains `stopped`?